core-bundle/src/Routing/RouteProvider.php

```diff
@@ -301,7 +301,7 @@ private function findRootPages(string $httpHost): array
             $rootPages = $pages->getModels();
         }
 
-        $pages = $pageModel->findBy(["tl_page.alias='index' OR tl_page.alias='/'"], null);
+        $pages = $pageModel->findAll(['column' => ["tl_page.alias='index' OR tl_page.alias='/'"]]);
```

Comment:
I like the original syntax more. Can't we preserve this kind of usage?

Reply:
I don’t think so because `null` can be an actual value that gets used by a `?` in the query.

How about the following?
```suggestion
        $pages = $pageModel->findBy(["tl_page.alias=? OR tl_page.alias=?"], ['index', '/']);
```

Reply:
```suggestion
        $pages = $pageModel->findBy(["tl_page.alias='index' OR tl_page.alias='/'"], []);
```
This could also work maybe (need to test :)

Reply:
The first one is a good compromise. 👍

Reply:
Fixed in 67faed35dd2b54a4b93e3cf7c9a05ab573023631